src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -263,13 +265,26 @@ private StatusResponsePayload ConvertFrom(DurableOrchestrationStatus status)
             };
         }
 
-        private static DateTime GetDateTimeQueryParameterValue(NameValueCollection queryStringNameValueCollection, string queryParameterName)
+        private static IEnumerable<OrchestrationRuntimeStatus> GetIEnumerableQueryParameterValue(NameValueCollection queryStringNameValueCollection, string queryParameterName)
         {
-            var value = queryStringNameValueCollection[queryParameterName];
-            DateTime.TryParse(value, out DateTime dateTime);
-            return dateTime;
+            var results = new List<OrchestrationRuntimeStatus>();
+            var parameters = queryStringNameValueCollection.GetValues(queryParameterName) ?? new string[] { };

```

Comment:
It looks like you are expecting multiple query string parameters with the same name - e.g. `runtimeStatus[]=foo&runtimeStatus[]=bar`. I do not recommend this design. Instead, I suggest you do it like this `runtimeStatus=foo,bar` - i.e. using a single query string parameter and separating multiple values using a comma. This is more natural, less verbose, and will be better supported across different software stacks. This code will then need to be changed to look for a single parameter value and split it at the comma.

Reply:
Totally make sense. I've done it.